api/cases/sscsCase.template.js

```diff
@@ -52,6 +52,15 @@ module.exports = {
                                     value: '$.case_data.panel.disabilityQualifiedMember'
                                 }
                             ]
+                        },
+                        {
+                            id: 'recentEvents',
```

Comment:
It would be better to introduce type properties rather than ids here

Reply:
did think that should I make Case Details a type to of simple list or somthing?

Reply:
Was thinking more that instead adding `id` you add a `type` property as used higher up the config to define what type of data it is. The summary sections could be `info-panel` and this one to `event-panel` or something

Reply:
```
            <ng-container *ngFor="let section of panelData.sections">
                <ng-container [ngSwitch]="section.type" >
                    <app-data-list
                        data-selector="data-list"
                        *ngSwitchCase="'data-list'"
                        [title]="section.name"
                        [dataList]="section.fields">
                    </app-data-list>

                    <app-timeline-section
                        data-selector="timeline-section"
                        *ngSwitchCase="'timeline'"
                        [title]="section.name"
                        [caseId]="caseId"
                        [events]="section.fields[0].value"
                    ></app-timeline-section>
                </ng-container>
            </ng-container>
```
still not happy but you can pick a type and pass the data that way